helpers/onfido.ts

```diff
@@ -1,9 +1,9 @@
-const { Onfido, Region } = require('@onfido/api');
+import { Onfido, Region } from '@onfido/api';
 
-const apiToken = process.env.SDK_TOKEN_FACTORY_SECRET;
+const apiToken = process.env.SDK_TOKEN_FACTORY_SECRET || '';
```

Comment:
You might want to use `??` instead of `||`. See https://github.com/NEARFoundation/tx-tracking-app/pull/13#discussion_r984628694